src/Tiles/Providers/RosreestrProvider.h

```diff
@@ -72,7 +72,7 @@ class RosreestrBordersProvider : public WmsMercatorProvider
 
 	CString MakeTileImageUrl(CPoint &pos, int zoom)
 	{
-		CString s = _urlFormat + GetBoundingBox(pos, zoom) + _url2;
+		CString s = _urlFormat + GetBoundingBox(pos, zoom, tkWmsVersion::wvAuto, tkWmsBoundingBoxOrder::bboAuto) + _url2;

```

Comment:
I don't see the change to the IDL for the new enumeration (tkWmsBoundingBoxOrder).  Nor for the the tkWmsVersion.wvAuto.

Reply:
I just committed the custom IDL.
wvAuto I have taken from tkWmsVersion (line 81)